### DISCUSSION TITLE
Customized geomap popups

### DISCUSSION THREAD

**Discussion by luizluca** [grafana/grafana#57856] [Dashboards, Panels & Visualizations]
In geomap, it would be really nice if we could select which fields appears on the tooltip. Some values, like lat and lon cannot be filtered in a trasnformation but just polute the visualization. The seletion should be different for each layer

**Comment by espressojunkie** [comment]
The Workmap does a much better job in the popups than Geomap ..

**Reply by robouden** [reply]
The workmap? Maybe you wanted to mention the Wolrdmap?

Regards
Rob Oudendijk

**Comment by james7272** [comment]
I suggest an HTML template, per layer, that has placeholders for query results and variables.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
